litmus-portal/frontend/src/pages/AnalyticsPage/styles.ts

```diff
@@ -41,13 +41,18 @@ const useStyles = makeStyles((theme) => ({
 
   waitingText: {
     fontSize: '2rem',
-    marginLeft: '25%',
-    marginTop: '25%',
+    marginLeft: '30%',
+    marginTop: '20%',
```

Comment:
Please try using theme.spacing() here.

Reply:
I tried this but have to add breakpoints then just for this loading message. If I hardcode the spacing it won't be as responsive as using `%` for the components which are required all have spacing from the theme itself, this is just an error message. @A4k9